test/sql.test.js

```diff
@@ -80,6 +99,12 @@ describe('sql connector', function() {
     expect(column).to.eql('LASTNAME');
   });
 
+  it('prefers database-specific column name over property name even the column name \
+  dose not follow the connector-specific configuration (UPPERCASE)', function() {
```

Comment:
nitpick: `even the` -> `even if the`
`dose` -> `does`

`uses` instead of `prefers`?